### ISSUE TITLE
The code in the script tag was mistakenly executed as a property error

### ISSUE THREAD

**Issue by hsuna** [nuejs/nue#183]
1. Create an HTML and enter the following code
```
<html>
  <head>
  </head>

  <body>
    <script type="text/javascript">
      var LG = {
        a: 1,
        b: 2
      }
    </script>
  </body>
</html>
```
2. Command  `nue`；

Output：
<img width="516" alt="image" src="https://github.com/nuejs/nue/assets/25148174/04c93eda-df99-4202-a265-48bb7f5d30dd">

Result: The code inside the script is garbled

**Comment by hsuna** [comment]
This issue occurred in version 0.3.1, which has been fixed in version 0.3.3

**Comment by nobkd** [comment]
Here's more information on what changed, if you want it:

https://github.com/nuejs/nue/issues/175#issuecomment-1905460135
